### DISCUSSION TITLE
Can anybody help with this code?

### DISCUSSION THREAD

**Discussion by Peter-Dobbins** [community/community#47594] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

There is a white gap at the top that I cannot get rid of. Can anybody help? Here is the HTML:

<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta name="description" content="Scott's Website" />
    <title>Scott's</title>
    <link rel="stylesheet" href="styles.css" />
  </head>
  <body class="space">
    <header>
      </div>
   </header>
  <main>
    <div class="writing">Opening Hours</div>
     </main>
 </body>
  </html>

Here is the CSS: 

::-webkit-scrollbar {
  display: none;
}
.space {
  margin: 0;
  margin-left: -1px;
  }
  .writing {
    font-size: 10rem;
    text-align: center;
    margin-top: 3rem;
  }
  img {
    border-bottom: 6px solid black;
    border-top: 6px solid black;
    display: block;
    margin-top: 0px;
    min-height: 100%;
    max-height: 100%;
  }
  
  header {
    margin-top: 0;
    padding-top: 0;
  }  
  body {
    padding-top: 0;
    margin-top: 0;
  }
  html {
    margin-top: 0;
    padding-top: 0;
  }

I can not put the html up here without it displaying only the image. Can anyone help?

**Comment by Bibiwei-Pere** [comment]
Hi @Peter-Dobbins,

The html code you provided above doesn't have a head and body tag. Please add the necessary tags and try using the following css code.

* {
  box-sizing: border-box;
}
body {
margin: 0;
}

Hope this helps.
